handheld/mesa-handheld/mesa-handheld/PKGBUILD

```diff
@@ -0,0 +1,362 @@
+        '8bc6f68ed286bea617a2cfaf3949bb699d3a0466faeca735314a51596ce950e4ee57eda88154bd562c1728cfaff4cdb5bc1ba701b9d47a9c50d4c4f011bee975')
+
+# https://docs.mesa3d.org/relnotes.html
+
+prepare() {
+  cd mesa-$pkgver
+
+  local src
+  for src in "${source[@]}"; do
+    src="${src%%::*}"
+    src="${src##*/}"
+    src="${src%.zst}"
+    [[ $src = *.patch ]] || continue
+    echo "Applying patch $src..."
+    patch -Np1 < "../$src"
+  done
+
+  # Include package release in version string so Chromium invalidates
+  # its GPU cache; otherwise it can cause pages to render incorrectly.
+  # https://bugs.launchpad.net/ubuntu/+source/chromium-browser/+bug/2020604
+  echo "$pkgver-cachyos$epoch.$pkgrel" >VERSION
+}
+
+build() {
+  local meson_options=(
+    -D android-libbacktrace=disabled
+    -D b_ndebug=true
+    -D gallium-drivers=radeonsi,swrast,iris,zink
+    -D gallium-extra-hud=true
+    -D gallium-rusticl=true
+    -D gallium-xa=disabled
+    -D gles1=disabled
+    -D html-docs=enabled
+    -D libunwind=disabled
+    -D microsoft-clc=disabled
+    -D valgrind=enabled
+    -D video-codecs=all
+    -D vulkan-drivers=amd,intel
+    -D vulkan-layers=device-select,intel-nullhw,overlay,screenshot,vram-report-limit
+  )
+
+  # Build only minimal debug info to reduce size
+  CFLAGS+=" -g1"
+  CXXFLAGS+=" -g1"
+
+  # Inject subproject packages
+  export MESON_PACKAGE_CACHE_DIR="$srcdir"
+
+  arch-meson mesa-$pkgver build "${meson_options[@]}"
+  meson compile -C build
+}
+
+_pick() {
+  local p="$1" f d; shift
+  for f; do
+    d="$srcdir/$p/${f#$pkgdir/}"
+    mkdir -p "$(dirname "$d")"
+    mv -v "$f" "$d"
+    rmdir -p --ignore-fail-on-non-empty "$(dirname "$f")"
+  done
+}
+
+package_mesa-handheld() {
+  depends=(
+    expat
+    gcc-libs
+    glibc
+    libdrm
+    libelf
+    libglvnd
+    libx11
+    libxcb
+    libxext
+    libxshmfence
+    libxxf86vm
+    llvm-libs
+    lm_sensors
+    spirv-tools
+    wayland
+    zlib
+    zstd
+  )
+  optdepends=("opengl-man-pages: for the OpenGL API man pages")
+  provides=(
+    "libva-mesa-driver=$epoch:$pkgver-$pkgrel"
+    "mesa-libgl=$epoch:$pkgver-$pkgrel"
+    "mesa-vdpau=$epoch:$pkgver-$pkgrel"
+    libva-driver
+    opengl-driver
+    vdpau-driver
+    mesa
+  )
+  conflicts=(
+    'libva-mesa-driver<1:24.2.7-1'
+    'mesa-libgl<17.0.1-2'
+    'mesa-vdpau<1:24.2.7-1'
+    mesa
+  )
+  replaces=(
+    'libva-mesa-driver<1:24.2.7-1'
+    'mesa-libgl<17.0.1-2'
+    'mesa-vdpau<1:24.2.7-1'
+    'mesa'
+  )
+
+  meson install -C build --destdir "$pkgdir" --no-rebuild
+
+  (
+    local libdir=usr/lib icddir=usr/share/vulkan/icd.d
+
+    cd "$pkgdir"
+
+    _pick opencl $libdir/libRusticlOpenCL*
+    _pick opencl etc/OpenCL/vendors/rusticl.icd
+
+    _pick vkgfxstr $icddir/gfxstream_vk_icd.*.json
+    _pick vkgfxstr $libdir/libvulkan_gfxstream.so
+
+    _pick vkintel $icddir/intel{,_hasvk}_icd.*.json
+    _pick vkintel $libdir/libvulkan_intel{,_hasvk}.so
+
+    _pick vknvidia $icddir/nouveau_icd.*.json
```

Comment:
Same as for lib32.

Reply:
are we talking specifically the `nouveau_icd.*.json`? or does that also mean the entirety of nouveau?